src/apic-extension/azext_apic_extension/aaz/latest/apic/api/deployment/_create.py

```diff
@@ -15,10 +15,10 @@
     "apic api deployment create",
 )
 class Create(AAZCommand):
-    """Create new or updates existing API deployment.
+    """Create a new API deployment or update an existing API deployment.
 
     :example: Create deployment
-        az apic api deployment create -g api-center-test -s contoso --name production --title "Production deployment" --description "Public cloud production deployment." --api echo-api --environmentId "/workspaces/default/environments/production" --definitionId "/workspaces/default/apis/echo-api/versions/2023-01-01/definitions/openapi"
+        az apic api deployment create -g api-center-test -s contoso --deployment-id production --title "Production deployment" --description "Public cloud production deployment." --api-id echo-api --environment-id "/workspaces/default/environments/production" --definition-id "/workspaces/default/apis/echo-api/versions/2023-01-01/definitions/openapi"
```

Comment:
Since custom properties is not new parameter, and previous version does not contain related example. I personally think it's not a P0 thing and prefer to add new examples later. Will let Julia to determine the priority.

Reply:
Will work with Julia to see what additional samples need to be added in the next version.